### ISSUE TITLE
[no-base-to-string] doesn't catch toLocaleString

### ISSUE THREAD

**Issue by danielnixon** [typescript-eslint/typescript-eslint#2440]
[no-base-to-string] doesn't catch toLocaleString

**Repro**

```TS
const foo = {};

console.log(foo.toLocaleString());
```

**Expected Result**

`toLocaleString` banned alongside `toString`

**Actual Result**

no lint error

**Comment by bradzacher** [comment]
This was just never added to the rule.
Happy to accept a PR!
